examples/mp-ping-pong/src/main.rs

```diff
@@ -277,6 +271,57 @@ fn test_mc_limited_message_drops(config: &TestConfig) -> TestResult {
     }
 }
 
+fn test_mc_consecutive_messages(config: &TestConfig) -> TestResult {
+    let system = build_system(config);
+    let data = r#"{"value": 0}"#.to_string();
+    let data2 = r#"{"value": 1}"#.to_string();
+
+    let messages_data = vec![data, data2];
+    let mut messages_prefix = Vec::new();
+    let mut collected_states = HashSet::new();
+
+    for (i, message_data) in messages_data.iter().enumerate() {
```

Comment:
чтобы не преобразовывать везде `i` в `u64`:

```suggestion
    for (message_data, i) in messages_data.iter().zip(0u64..) {
```

Reply:
И можно даже стартовать с 1, чтобы избавиться от `i+1`